plugins/doc_fragments/pulp.py

```diff
@@ -38,6 +36,12 @@ class ModuleDocFragment(object):
       - If no value is specified, the value of the environment variable C(SQUEEZER_VALIDATE_CERTS) will be used as a fallback.
     type: bool
     default: true
+  refresh_api_cache:
+    description:
+      - Whether the cached api documentation should be invalidated.
```

Comment:
```suggestion
      - Whether the cached API documentation should be invalidated.
```

also maybe "specification" instead of "documentation"?

Reply:
```suggestion
      - Whether the cached API specification should be invalidated.
```